### ISSUE TITLE
No module named matplotlib

### ISSUE THREAD

**Issue by Jana-Wessels** [tensorflow/models#3703]
I am having an issue when running the training script for the pets tutorial, it is working locally but don't have the computing capabilities, very time I run it gives the issue of no module named matplotlib. I have already done the fixes mentioned in #3607 and it didn't help.

Please help I am getting so hopeless.

Traceback (most recent call last): File "/usr/lib/python3.5/runpy.py", line 184, in _run_module_as_main "__main__", mod_spec) File "/usr/lib/python3.5/runpy.py", line 85, in _run_code exec(code, run_globals) File "/root/.local/lib/python3.5/site-packages/object_detection/train.py", line 51, in <module> from object_detection.builders import model_builder File "/root/.local/lib/python3.5/site-packages/object_detection/builders/model_builder.py", line 29, in <module> from object_detection.meta_architectures import ssd_meta_arch File "/root/.local/lib/python3.5/site-packages/object_detection/meta_architectures/ssd_meta_arch.py", line 32, in <module> from object_detection.utils import visualization_utils File "/root/.local/lib/python3.5/site-packages/object_detection/utils/visualization_utils.py", line 25, in <module> import matplotlib; matplotlib.use('Agg') # pylint: disable=multiple-statements ImportError: No module named 'matplotlib'

**my setup.py is as follows:**
"""Setup script for object_detection."""
import logging
import subprocess
from setuptools import find_packages
from setuptools import setup
from setuptools.command.install import install

class CustomCommands(install):
  """A setuptools Command class able to run arbitrary commands."""

def RunCustomCommand(self, command_list):
    p = subprocess.Popen(
        command_list,
        stdin=subprocess.PIPE,
        stdout=subprocess.PIPE,
        stderr=subprocess.STDOUT)
    # Can use communicate(input='y\n'.encode()) if the command run requires
    # some confirmation.
    stdout_data, _ = p.communicate()
    logging.info('Log command output: %s', stdout_data)
    if p.returncode != 0:
      raise RuntimeError('Command %s failed: exit code: %s' %
                         (command_list, p.returncode))
       
  def run(self):
    self.RunCustomCommand(['apt-get', 'update'])
    self.RunCustomCommand(
          ['apt-get', 'install', '-y', 'python-tk'])

    install.run(self)

REQUIRED_PACKAGES = ['Pillow>=1.0', 'Matplotlib>=2.1']

setup(
    name='object_detection',
    version='0.1',
    install_requires=REQUIRED_PACKAGES,
    include_package_data=True,
    packages=[p for p in find_packages() if p.startswith('object_detection')],
    description='Tensorflow Object Detection Library',
 cmdclass={
        'install': CustomCommands,
    }
)

**cloud.yml:**
trainingInput:
  runtimeVersion: "1.5"
  scaleTier: CUSTOM
  masterType: standard_gpu
  workerCount: 5
  workerType: standard_gpu
  parameterServerCount: 3
  parameterServerType: standard
  pythonVersion: "3.5"

### System information
- **What is the top-level directory of the model you are using**: gs://pets_jana
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**:
Modified based on #3607 
- **OS Platform and Distribution**: Linux 
- **TensorFlow installed from (source or binary)**: source
- **TensorFlow version (use command below)**: 1.6.0
- **GPU model and memory**: cpu

Tensorflow = 1.6
Python = 3.6.4

### Describe the problem
I have tried all the fixes I could see here and none of them changed the issue of no module named matplotlib. I have no idea where this issue is coming from.

**Comment by wkoenig365** [comment]
Please verify that you have installed matplotlib for python 3.x as a user module. Try:

`pip3 install --user matplotlib`

to verify matplotlib is indeed installed for python 3.x. If you need it globally, remove the user flag. If you are running your scripts as root, make sure to execute pip3 with root privileges (sudo pip3). Otherwise you should be able to install matplotlib through apt-get

`apt-get install python3-matplotlib`

Please also check the output of:

`pip3 show matplotlib`

or if you installed with apt-get:

`dpkg -L python3-matplotlib`

to make sure it is installed in the right directory (looks like it should be something like '/root/.local/lib/python3.5/site-packages/' in your case).

**Comment by Jana-Wessels** [comment]
This is the output of the commands , I ran the training again and it had the same error:

The replica ps 0 exited with a non-zero status of 1. Termination reason: Error. Traceback (most recent call last): File "/usr/lib/python3.5/runpy.py", line 184, in _run_module_as_main "__main__", mod_spec) File "/usr/lib/python3.5/runpy.py", line 85, in _run_code exec(code, run_globals) File "/root/.local/lib/python3.5/site-packages/object_detection/train.py", line 51, in <module> from object_detection.builders import model_builder File "/root/.local/lib/python3.5/site-packages/object_detection/builders/model_builder.py", line 29, in <module> from object_detection.meta_architectures import ssd_meta_arch File "/root/.local/lib/python3.5/site-packages/object_detection/meta_architectures/ssd_meta_arch.py", line 32, in <module> from object_detection.utils import visualization_utils File "/root/.local/lib/python3.5/site-packages/object_detection/utils/visualization_utils.py", line 25, in <module> import matplotlib; matplotlib.use('Agg') # pylint: disable=multiple-statements ImportError: No module named 'matplotlib' The replica ps 1 exited with a non-zero status of 1. Termination reason: Error. Traceback (most recent call last): File "/usr/lib/python3.5/runpy.py", line 184, in _run_module_as_main "__main__", mod_spec) File "/usr/lib/python3.5/runpy.py", line 85, in _run_code exec(code, run_globals) File "/root/.local/lib/python3.5/site-packages/object_detection/train.py", line 51, in <module> from object_detection.builders import model_builder File "/root/.local/lib/python3.5/site-packages/object_detection/builders/model_builder.py", line 29, in <module> from object_detection.meta_architectures import ssd_meta_arch File "/root/.local/lib/python3.5/site-packages/object_detection/meta_architectures/ssd_meta_arch.py", line 32, in <module> from object_detection.utils import visualization_utils File "/root/.local/lib/python3.5/site-packages/object_detection/utils/visualization_utils.py", line 25, in <module> import matplotlib; matplotlib.use('Agg') # pylint: disable=multiple-statements ImportError: No module named 'matplotlib' The replica ps 2 exited with a non-zero status of 1. Termination reason: Error. Traceback (most recent call last): File "/usr/lib/python3.5/runpy.py", line 184, in _run_module_as_main "__main__", mod_spec) File "/usr/lib/python3.5/runpy.py", line 85, in _run_code exec(code, run_globals) File "/root/.local/lib/python3.5/site-packages/object_detection/train.py", line 51, in <module> from object_detection.builders import model_builder File "/root/.local/lib/python3.5/site-packages/object_detection/builders/model_builder.py", line 29, in <module> from object_detection.meta_architectures import ssd_meta_arch File "/root/.local/lib/python3.5/site-packages/object_detection/meta_architectures/ssd_meta_arch.py", line 32, in <module> from object_detection.utils import visualization_utils File "/root/.local/lib/python3.5/site-packages/object_detection/utils/visualization_utils.py", line 25, in <module> import matplotlib; matplotlib.use('Agg') # pylint: disable=multiple-statements ImportError: No module named 'matplotlib' To find out more about why your job exited please check the logs: https://console.cloud.google.com/logs/viewer?project=827587257014&resource=ml_job%2Fjob_id%2Fjana_object_detection_1521788940&advancedFilter=resource.type%3D%22ml_job%22%0Aresource.labels.job_id%3D%22jana_object_detection_1521788940%22 




**pip3 show matplotlib**
Name: matplotlib
Version: 2.1.0
Summary: Python plotting package
Home-page: http://matplotlib.org
Author: John D. Hunter, Michael Droettboom
Author-email: matplotlib-users@python.org
License: BSD
Location: /home/jana/anaconda3/lib/python3.6/site-packages
Requires: numpy, six, python-dateutil, pytz, cycler, pyparsing


 **dpkg -L python3-matplotlib**
/.
/usr
/usr/lib
/usr/lib/python3
/usr/lib/python3/dist-packages
/usr/lib/python3/dist-packages/mpl_toolkits
/usr/lib/python3/dist-packages/mpl_toolkits/__init__.py
/usr/lib/python3/dist-packages/mpl_toolkits/axisartist
/usr/lib/python3/dist-packages/mpl_toolkits/axisartist/angle_helper.py
/usr/lib/python3/dist-packages/mpl_toolkits/axisartist/__init__.py
/usr/lib/python3/dist-packages/mpl_toolkits/axisartist/clip_path.py
/usr/lib/python3/dist-packages/mpl_toolkits/axisartist/axisline_style.py
/usr/lib/python3/dist-packages/mpl_toolkits/axisartist/grid_finder.py
/usr/lib/python3/dist-packages/mpl_toolkits/axisartist/axislines.py
/usr/lib/python3/dist-packages/mpl_toolkits/axisartist/grid_helper_curvelinear.py
/usr/lib/python3/dist-packages/mpl_toolkits/axisartist/axis_artist.py
/usr/lib/python3/dist-packages/mpl_toolkits/axisartist/floating_axes.py
/usr/lib/python3/dist-packages/mpl_toolkits/gtktools.py
/usr/lib/python3/dist-packages/mpl_toolkits/axes_grid1
/usr/lib/python3/dist-packages/mpl_toolkits/axes_grid1/axes_size.py
/usr/lib/python3/dist-packages/mpl_toolkits/axes_grid1/inset_locator.py
/usr/lib/python3/dist-packages/mpl_toolkits/axes_grid1/axes_rgb.py
/usr/lib/python3/dist-packages/mpl_toolkits/axes_grid1/mpl_axes.py
/usr/lib/python3/dist-packages/mpl_toolkits/axes_grid1/__init__.py
/usr/lib/python3/dist-packages/mpl_toolkits/axes_grid1/axes_divider.py
/usr/lib/python3/dist-packages/mpl_toolkits/axes_grid1/colorbar.py
/usr/lib/python3/dist-packages/mpl_toolkits/axes_grid1/axes_grid.py
/usr/lib/python3/dist-packages/mpl_toolkits/axes_grid1/parasite_axes.py
/usr/lib/python3/dist-packages/mpl_toolkits/axes_grid1/anchored_artists.py
/usr/lib/python3/dist-packages/mpl_toolkits/axes_grid
/usr/lib/python3/dist-packages/mpl_toolkits/axes_grid/axes_size.py
/usr/lib/python3/dist-packages/mpl_toolkits/axes_grid/angle_helper.py
/usr/lib/python3/dist-packages/mpl_toolkits/axes_grid/inset_locator.py
/usr/lib/python3/dist-packages/mpl_toolkits/axes_grid/axes_rgb.py
/usr/lib/python3/dist-packages/mpl_toolkits/axes_grid/__init__.py
/usr/lib/python3/dist-packages/mpl_toolkits/axes_grid/clip_path.py
/usr/lib/python3/dist-packages/mpl_toolkits/axes_grid/axisline_style.py
/usr/lib/python3/dist-packages/mpl_toolkits/axes_grid/axes_divider.py
/usr/lib/python3/dist-packages/mpl_toolkits/axes_grid/colorbar.py
/usr/lib/python3/dist-packages/mpl_toolkits/axes_grid/axes_grid.py
/usr/lib/python3/dist-packages/mpl_toolkits/axes_grid/parasite_axes.py
/usr/lib/python3/dist-packages/mpl_toolkits/axes_grid/grid_finder.py
/usr/lib/python3/dist-packages/mpl_toolkits/axes_grid/anchored_artists.py
/usr/lib/python3/dist-packages/mpl_toolkits/axes_grid/axislines.py
/usr/lib/python3/dist-packages/mpl_toolkits/axes_grid/grid_helper_curvelinear.py
/usr/lib/python3/dist-packages/mpl_toolkits/axes_grid/axis_artist.py
/usr/lib/python3/dist-packages/mpl_toolkits/axes_grid/floating_axes.py
/usr/lib/python3/dist-packages/mpl_toolkits/tests
/usr/lib/python3/dist-packages/mpl_toolkits/tests/test_mplot3d.py
/usr/lib/python3/dist-packages/mpl_toolkits/tests/test_axes_grid1.py
/usr/lib/python3/dist-packages/mpl_toolkits/tests/baseline_images
/usr/lib/python3/dist-packages/mpl_toolkits/tests/baseline_images/test_axes_grid
/usr/lib/python3/dist-packages/mpl_toolkits/tests/baseline_images/test_axes_grid/imagegrid_cbar_mode.png
/usr/lib/python3/dist-packages/mpl_toolkits/tests/baseline_images/test_axes_grid1
/usr/lib/python3/dist-packages/mpl_toolkits/tests/baseline_images/test_axes_grid1/divider_append_axes.pdf
/usr/lib/python3/dist-packages/mpl_toolkits/tests/baseline_images/test_axes_grid1/divider_append_axes.svg
/usr/lib/python3/dist-packages/mpl_toolkits/tests/baseline_images/test_axes_grid1/divider_append_axes.png
/usr/lib/python3/dist-packages/mpl_toolkits/tests/baseline_images/test_mplot3d
/usr/lib/python3/dist-packages/mpl_toolkits/tests/baseline_images/test_mplot3d/wireframe3d.pdf
/usr/lib/python3/dist-packages/mpl_toolkits/tests/baseline_images/test_mplot3d/quiver3d_masked.svg
/usr/lib/python3/dist-packages/mpl_toolkits/tests/baseline_images/test_mplot3d/trisurf3d.png
/usr/lib/python3/dist-packages/mpl_toolkits/tests/baseline_images/test_mplot3d/contourf3d_fill.svg
/usr/lib/python3/dist-packages/mpl_toolkits/tests/baseline_images/test_mplot3d/lines3d.png
/usr/lib/python3/dist-packages/mpl_toolkits/tests/baseline_images/test_mplot3d/scatter3d.png
/usr/lib/python3/dist-packages/mpl_toolkits/tests/baseline_images/test_mplot3d/surface3d.png
/usr/lib/python3/dist-packages/mpl_toolkits/tests/baseline_images/test_mplot3d/quiver3d.pdf
/usr/lib/python3/dist-packages/mpl_toolkits/tests/baseline_images/test_mplot3d/trisurf3d.svg
/usr/lib/python3/dist-packages/mpl_toolkits/tests/baseline_images/test_mplot3d/contourf3d_fill.png
/usr/lib/python3/dist-packages/mpl_toolkits/tests/baseline_images/test_mplot3d/wireframe3dzerorstride.png
/usr/lib/python3/dist-packages/mpl_toolkits/tests/baseline_images/test_mplot3d/quiver3d.svg
/usr/lib/python3/dist-packages/mpl_toolkits/tests/baseline_images/test_mplot3d/mixedsubplot.png
/usr/lib/python3/dist-packages/mpl_toolkits/tests/baseline_images/test_mplot3d/lines3d.pdf
/usr/lib/python3/dist-packages/mpl_toolkits/tests/baseline_images/test_mplot3d/bar3d.svg
/usr/lib/python3/dist-packages/mpl_toolkits/tests/baseline_images/test_mplot3d/contour3d.svg
/usr/lib/python3/dist-packages/mpl_toolkits/tests/baseline_images/test_mplot3d/quiver3d.png
/usr/lib/python3/dist-packages/mpl_toolkits/tests/baseline_images/test_mplot3d/quiver3d_pivot_tail.png
/usr/lib/python3/dist-packages/mpl_toolkits/tests/baseline_images/test_mplot3d/mixedsubplot.svg
/usr/lib/python3/dist-packages/mpl_toolkits/tests/baseline_images/test_mplot3d/bar3d.png
/usr/lib/python3/dist-packages/mpl_toolkits/tests/baseline_images/test_mplot3d/text3d.svg
/usr/lib/python3/dist-packages/mpl_toolkits/tests/baseline_images/test_mplot3d/contourf3d.png
/usr/lib/python3/dist-packages/mpl_toolkits/tests/baseline_images/test_mplot3d/quiver3d_empty.pdf
/usr/lib/python3/dist-packages/mpl_toolkits/tests/baseline_images/test_mplot3d/lines3d.svg
/usr/lib/python3/dist-packages/mpl_toolkits/tests/baseline_images/test_mplot3d/quiver3d_masked.pdf
/usr/lib/python3/dist-packages/mpl_toolkits/tests/baseline_images/test_mplot3d/wireframe3dzerocstride.png
/usr/lib/python3/dist-packages/mpl_toolkits/tests/baseline_images/test_mplot3d/contourf3d_fill.pdf
/usr/lib/python3/dist-packages/mpl_toolkits/tests/baseline_images/test_mplot3d/quiver3d_empty.svg
/usr/lib/python3/dist-packages/mpl_toolkits/tests/baseline_images/test_mplot3d/mixedsubplot.pdf
/usr/lib/python3/dist-packages/mpl_toolkits/tests/baseline_images/test_mplot3d/text3d.png
/usr/lib/python3/dist-packages/mpl_toolkits/tests/baseline_images/test_mplot3d/axes3d_cla.png
/usr/lib/python3/dist-packages/mpl_toolkits/tests/baseline_images/test_mplot3d/quiver3d_empty.png
/usr/lib/python3/dist-packages/mpl_toolkits/tests/baseline_images/test_mplot3d/contourf3d.svg
/usr/lib/python3/dist-packages/mpl_toolkits/tests/baseline_images/test_mplot3d/surface3d.svg
/usr/lib/python3/dist-packages/mpl_toolkits/tests/baseline_images/test_mplot3d/wireframe3d.svg
/usr/lib/python3/dist-packages/mpl_toolkits/tests/baseline_images/test_mplot3d/quiver3d_masked.png
/usr/lib/python3/dist-packages/mpl_toolkits/tests/baseline_images/test_mplot3d/scatter3d.pdf
/usr/lib/python3/dist-packages/mpl_toolkits/tests/baseline_images/test_mplot3d/contour3d.png
/usr/lib/python3/dist-packages/mpl_toolkits/tests/baseline_images/test_mplot3d/quiver3d_pivot_middle.png
/usr/lib/python3/dist-packages/mpl_toolkits/tests/baseline_images/test_mplot3d/axes3d_labelpad.png
/usr/lib/python3/dist-packages/mpl_toolkits/tests/baseline_images/test_mplot3d/scatter3d.svg
/usr/lib/python3/dist-packages/mpl_toolkits/tests/baseline_images/test_mplot3d/bar3d.pdf
/usr/lib/python3/dist-packages/mpl_toolkits/tests/baseline_images/test_mplot3d/contourf3d.pdf
/usr/lib/python3/dist-packages/mpl_toolkits/tests/baseline_images/test_mplot3d/surface3d.pdf
/usr/lib/python3/dist-packages/mpl_toolkits/tests/baseline_images/test_mplot3d/contour3d.pdf
/usr/lib/python3/dist-packages/mpl_toolkits/tests/baseline_images/test_mplot3d/wireframe3d.png
/usr/lib/python3/dist-packages/mpl_toolkits/tests/baseline_images/test_mplot3d/text3d.pdf
/usr/lib/python3/dist-packages/mpl_toolkits/tests/baseline_images/test_mplot3d/trisurf3d.pdf
/usr/lib/python3/dist-packages/mpl_toolkits/tests/__init__.py
/usr/lib/python3/dist-packages/mpl_toolkits/tests/test_axes_grid.py
/usr/lib/python3/dist-packages/mpl_toolkits/mplot3d
/usr/lib/python3/dist-packages/mpl_toolkits/mplot3d/proj3d.py
/usr/lib/python3/dist-packages/mpl_toolkits/mplot3d/__init__.py
/usr/lib/python3/dist-packages/mpl_toolkits/mplot3d/axis3d.py
/usr/lib/python3/dist-packages/mpl_toolkits/mplot3d/axes3d.py
/usr/lib/python3/dist-packages/mpl_toolkits/mplot3d/art3d.py
/usr/lib/python3/dist-packages/mpl_toolkits/exceltools.py
/usr/lib/python3/dist-packages/pylab.py
/usr/lib/python3/dist-packages/matplotlib-1.5.1-nspkg.pth
/usr/lib/python3/dist-packages/matplotlib-1.5.1.egg-info
/usr/lib/python3/dist-packages/matplotlib-1.5.1.egg-info/dependency_links.txt
/usr/lib/python3/dist-packages/matplotlib-1.5.1.egg-info/top_level.txt
/usr/lib/python3/dist-packages/matplotlib-1.5.1.egg-info/PKG-INFO
/usr/lib/python3/dist-packages/matplotlib-1.5.1.egg-info/requires.txt
/usr/lib/python3/dist-packages/matplotlib-1.5.1.egg-info/not-zip-safe
/usr/lib/python3/dist-packages/matplotlib-1.5.1.egg-info/namespace_packages.txt
/usr/lib/python3/dist-packages/matplotlib
/usr/lib/python3/dist-packages/matplotlib/cm.py
/usr/lib/python3/dist-packages/matplotlib/cbook.py
/usr/lib/python3/dist-packages/matplotlib/backends
/usr/lib/python3/dist-packages/matplotlib/backends/backend_gtk3.py
/usr/lib/python3/dist-packages/matplotlib/backends/backend_svg.py
/usr/lib/python3/dist-packages/matplotlib/backends/backend_gtk3agg.py
/usr/lib/python3/dist-packages/matplotlib/backends/_tkagg.cpython-35m-x86_64-linux-gnu.so
/usr/lib/python3/dist-packages/matplotlib/backends/backend_nbagg.py
/usr/lib/python3/dist-packages/matplotlib/backends/backend_qt5agg.py
/usr/lib/python3/dist-packages/matplotlib/backends/backend_webagg_core.py
/usr/lib/python3/dist-packages/matplotlib/backends/backend_template.py
/usr/lib/python3/dist-packages/matplotlib/backends/backend_wx.py
/usr/lib/python3/dist-packages/matplotlib/backends/__init__.py
/usr/lib/python3/dist-packages/matplotlib/backends/backend_tkagg.py
/usr/lib/python3/dist-packages/matplotlib/backends/tkagg.py
/usr/lib/python3/dist-packages/matplotlib/backends/backend_cocoaagg.py
/usr/lib/python3/dist-packages/matplotlib/backends/wx_compat.py
/usr/lib/python3/dist-packages/matplotlib/backends/backend_webagg.py
/usr/lib/python3/dist-packages/matplotlib/backends/web_backend
/usr/lib/python3/dist-packages/matplotlib/backends/web_backend/all_figures.html
/usr/lib/python3/dist-packages/matplotlib/backends/web_backend/single_figure.html
/usr/lib/python3/dist-packages/matplotlib/backends/web_backend/nbagg_uat.ipynb
/usr/lib/python3/dist-packages/matplotlib/backends/web_backend/mpl.js
/usr/lib/python3/dist-packages/matplotlib/backends/web_backend/ipython_inline_figure.html
/usr/lib/python3/dist-packages/matplotlib/backends/web_backend/mpl_tornado.js
/usr/lib/python3/dist-packages/matplotlib/backends/web_backend/nbagg_mpl.js
/usr/lib/python3/dist-packages/matplotlib/backends/web_backend/css
/usr/lib/python3/dist-packages/matplotlib/backends/web_backend/css/page.css
/usr/lib/python3/dist-packages/matplotlib/backends/web_backend/css/fbm.css
/usr/lib/python3/dist-packages/matplotlib/backends/web_backend/css/boilerplate.css
/usr/lib/python3/dist-packages/matplotlib/backends/web_backend/jquery
/usr/lib/python3/dist-packages/matplotlib/backends/web_backend/jquery/js
/usr/lib/python3/dist-packages/matplotlib/backends/web_backend/jquery/js/jquery-1.11.3.min.js
/usr/lib/python3/dist-packages/matplotlib/backends/web_backend/jquery/css
/usr/lib/python3/dist-packages/matplotlib/backends/web_backend/jquery/css/themes
/usr/lib/python3/dist-packages/matplotlib/backends/web_backend/jquery/css/themes/base
/usr/lib/python3/dist-packages/matplotlib/backends/web_backend/jquery/css/themes/base/images
/usr/lib/python3/dist-packages/matplotlib/backends/web_backend/jquery/css/themes/base/images/ui-icons_228ef1_256x240.png
/usr/lib/python3/dist-packages/matplotlib/backends/web_backend/jquery/css/themes/base/images/ui-icons_222222_256x240.png
/usr/lib/python3/dist-packages/matplotlib/backends/web_backend/jquery/css/themes/base/images/ui-bg_gloss-wave_35_f6a828_500x100.png
/usr/lib/python3/dist-packages/matplotlib/backends/web_backend/jquery/css/themes/base/images/ui-bg_diagonals-thick_18_b81900_40x40.png
/usr/lib/python3/dist-packages/matplotlib/backends/web_backend/jquery/css/themes/base/images/ui-bg_highlight-soft_100_eeeeee_1x100.png
/usr/lib/python3/dist-packages/matplotlib/backends/web_backend/jquery/css/themes/base/images/ui-bg_glass_100_f6f6f6_1x400.png
/usr/lib/python3/dist-packages/matplotlib/backends/web_backend/jquery/css/themes/base/images/ui-icons_ffffff_256x240.png
/usr/lib/python3/dist-packages/matplotlib/backends/web_backend/jquery/css/themes/base/images/ui-bg_highlight-soft_75_ffe45c_1x100.png
/usr/lib/python3/dist-packages/matplotlib/backends/web_backend/jquery/css/themes/base/images/ui-bg_glass_100_fdf5ce_1x400.png
/usr/lib/python3/dist-packages/matplotlib/backends/web_backend/jquery/css/themes/base/images/ui-bg_diagonals-thick_20_666666_40x40.png
/usr/lib/python3/dist-packages/matplotlib/backends/web_backend/jquery/css/themes/base/images/ui-icons_ef8c08_256x240.png
/usr/lib/python3/dist-packages/matplotlib/backends/web_backend/jquery/css/themes/base/images/ui-bg_glass_65_ffffff_1x400.png
/usr/lib/python3/dist-packages/matplotlib/backends/web_backend/jquery/css/themes/base/images/ui-icons_ffd27a_256x240.png
/usr/lib/python3/dist-packages/matplotlib/backends/web_backend/jquery/css/themes/base/images/ui-bg_flat_10_000000_40x100.png
/usr/lib/python3/dist-packages/matplotlib/backends/backend_wxagg.py
/usr/lib/python3/dist-packages/matplotlib/backends/qt_compat.py
/usr/lib/python3/dist-packages/matplotlib/backends/backend_qt4.py
/usr/lib/python3/dist-packages/matplotlib/backends/_backend_agg.cpython-35m-x86_64-linux-gnu.so
/usr/lib/python3/dist-packages/matplotlib/backends/qt4_compat.py
/usr/lib/python3/dist-packages/matplotlib/backends/backend_qt5.py
/usr/lib/python3/dist-packages/matplotlib/backends/backend_gtkcairo.py
/usr/lib/python3/dist-packages/matplotlib/backends/backend_gtk.py
/usr/lib/python3/dist-packages/matplotlib/backends/backend_macosx.py
/usr/lib/python3/dist-packages/matplotlib/backends/backend_pdf.py
/usr/lib/python3/dist-packages/matplotlib/backends/backend_qt4agg.py
/usr/lib/python3/dist-packages/matplotlib/backends/backend_gdk.py
/usr/lib/python3/dist-packages/matplotlib/backends/backend_gtkagg.py
/usr/lib/python3/dist-packages/matplotlib/backends/backend_ps.py
/usr/lib/python3/dist-packages/matplotlib/backends/backend_agg.py
/usr/lib/python3/dist-packages/matplotlib/backends/backend_mixed.py
/usr/lib/python3/dist-packages/matplotlib/backends/qt_editor
/usr/lib/python3/dist-packages/matplotlib/backends/qt_editor/formlayout.py
/usr/lib/python3/dist-packages/matplotlib/backends/qt_editor/__init__.py
/usr/lib/python3/dist-packages/matplotlib/backends/qt_editor/formsubplottool.py
/usr/lib/python3/dist-packages/matplotlib/backends/qt_editor/figureoptions.py
/usr/lib/python3/dist-packages/matplotlib/backends/windowing.py
/usr/lib/python3/dist-packages/matplotlib/backends/backend_gtk3cairo.py
/usr/lib/python3/dist-packages/matplotlib/backends/backend_cairo.py
/usr/lib/python3/dist-packages/matplotlib/backends/backend_pgf.py
/usr/lib/python3/dist-packages/matplotlib/animation.py
/usr/lib/python3/dist-packages/matplotlib/externals
/usr/lib/python3/dist-packages/matplotlib/externals/__init__.py
/usr/lib/python3/dist-packages/matplotlib/externals/six.py
/usr/lib/python3/dist-packages/matplotlib/path.py
/usr/lib/python3/dist-packages/matplotlib/markers.py
/usr/lib/python3/dist-packages/matplotlib/hatch.py
/usr/lib/python3/dist-packages/matplotlib/style
/usr/lib/python3/dist-packages/matplotlib/style/__init__.py
/usr/lib/python3/dist-packages/matplotlib/style/core.py
/usr/lib/python3/dist-packages/matplotlib/axes
/usr/lib/python3/dist-packages/matplotlib/axes/__init__.py
/usr/lib/python3/dist-packages/matplotlib/axes/_base.py
/usr/lib/python3/dist-packages/matplotlib/axes/_axes.py
/usr/lib/python3/dist-packages/matplotlib/axes/_subplots.py
/usr/lib/python3/dist-packages/matplotlib/_path.cpython-35m-x86_64-linux-gnu.so
/usr/lib/python3/dist-packages/matplotlib/textpath.py
/usr/lib/python3/dist-packages/matplotlib/table.py
/usr/lib/python3/dist-packages/matplotlib/_cm.py
/usr/lib/python3/dist-packages/matplotlib/docstring.py
/usr/lib/python3/dist-packages/matplotlib/widgets.py
/usr/lib/python3/dist-packages/matplotlib/gridspec.py
/usr/lib/python3/dist-packages/matplotlib/__init__.py
/usr/lib/python3/dist-packages/matplotlib/_version.py
/usr/lib/python3/dist-packages/matplotlib/stackplot.py
/usr/lib/python3/dist-packages/matplotlib/pylab.py
/usr/lib/python3/dist-packages/matplotlib/dviread.py
/usr/lib/python3/dist-packages/matplotlib/transforms.py
/usr/lib/python3/dist-packages/matplotlib/lines.py
/usr/lib/python3/dist-packages/matplotlib/tri
/usr/lib/python3/dist-packages/matplotlib/tri/triinterpolate.py
/usr/lib/python3/dist-packages/matplotlib/tri/tricontour.py
/usr/lib/python3/dist-packages/matplotlib/tri/__init__.py
/usr/lib/python3/dist-packages/matplotlib/tri/tritools.py
/usr/lib/python3/dist-packages/matplotlib/tri/triplot.py
/usr/lib/python3/dist-packages/matplotlib/tri/triangulation.py
/usr/lib/python3/dist-packages/matplotlib/tri/trifinder.py
/usr/lib/python3/dist-packages/matplotlib/tri/tripcolor.py
/usr/lib/python3/dist-packages/matplotlib/tri/trirefine.py
/usr/lib/python3/dist-packages/matplotlib/ttconv.cpython-35m-x86_64-linux-gnu.so
/usr/lib/python3/dist-packages/matplotlib/ticker.py
/usr/lib/python3/dist-packages/matplotlib/bezier.py
/usr/lib/python3/dist-packages/matplotlib/_tri.cpython-35m-x86_64-linux-gnu.so
/usr/lib/python3/dist-packages/matplotlib/contour.py
/usr/lib/python3/dist-packages/matplotlib/_cm_listed.py
/usr/lib/python3/dist-packages/matplotlib/_cntr.cpython-35m-x86_64-linux-gnu.so
/usr/lib/python3/dist-packages/matplotlib/legend.py
/usr/lib/python3/dist-packages/matplotlib/pyplot.py
/usr/lib/python3/dist-packages/matplotlib/projections
/usr/lib/python3/dist-packages/matplotlib/projections/__init__.py
/usr/lib/python3/dist-packages/matplotlib/projections/geo.py
/usr/lib/python3/dist-packages/matplotlib/projections/polar.py
/usr/lib/python3/dist-packages/matplotlib/backend_managers.py
/usr/lib/python3/dist-packages/matplotlib/tight_bbox.py
/usr/lib/python3/dist-packages/matplotlib/testing
/usr/lib/python3/dist-packages/matplotlib/testing/disable_internet.py
/usr/lib/python3/dist-packages/matplotlib/testing/__init__.py
/usr/lib/python3/dist-packages/matplotlib/testing/jpl_units
/usr/lib/python3/dist-packages/matplotlib/testing/jpl_units/UnitDblFormatter.py
/usr/lib/python3/dist-packages/matplotlib/testing/jpl_units/Duration.py
/usr/lib/python3/dist-packages/matplotlib/testing/jpl_units/Epoch.py
/usr/lib/python3/dist-packages/matplotlib/testing/jpl_units/__init__.py
/usr/lib/python3/dist-packages/matplotlib/testing/jpl_units/UnitDbl.py
/usr/lib/python3/dist-packages/matplotlib/testing/jpl_units/UnitDblConverter.py
/usr/lib/python3/dist-packages/matplotlib/testing/jpl_units/StrConverter.py
/usr/lib/python3/dist-packages/matplotlib/testing/jpl_units/EpochConverter.py
/usr/lib/python3/dist-packages/matplotlib/testing/exceptions.py
/usr/lib/python3/dist-packages/matplotlib/testing/decorators.py
/usr/lib/python3/dist-packages/matplotlib/testing/compare.py
/usr/lib/python3/dist-packages/matplotlib/testing/noseclasses.py
/usr/lib/python3/dist-packages/matplotlib/_image.cpython-35m-x86_64-linux-gnu.so
/usr/lib/python3/dist-packages/matplotlib/colorbar.py
/usr/lib/python3/dist-packages/matplotlib/texmanager.py
/usr/lib/python3/dist-packages/matplotlib/mathtext.py
/usr/lib/python3/dist-packages/matplotlib/units.py
/usr/lib/python3/dist-packages/matplotlib/offsetbox.py
/usr/lib/python3/dist-packages/matplotlib/compat
/usr/lib/python3/dist-packages/matplotlib/compat/__init__.py
/usr/lib/python3/dist-packages/matplotlib/compat/subprocess.py
/usr/lib/python3/dist-packages/matplotlib/_pylab_helpers.py
/usr/lib/python3/dist-packages/matplotlib/legend_handler.py
/usr/lib/python3/dist-packages/matplotlib/_delaunay.cpython-35m-x86_64-linux-gnu.so
/usr/lib/python3/dist-packages/matplotlib/tests
/usr/lib/python3/dist-packages/matplotlib/tests/test_compare_images.py
/usr/lib/python3/dist-packages/matplotlib/tests/test_contour.py
/usr/lib/python3/dist-packages/matplotlib/tests/test_bbox_tight.py
/usr/lib/python3/dist-packages/matplotlib/tests/test_streamplot.py
/usr/lib/python3/dist-packages/matplotlib/tests/test_figure.py
/usr/lib/python3/dist-packages/matplotlib/tests/test_font_manager.py
/usr/lib/python3/dist-packages/matplotlib/tests/test_dates.py
/usr/lib/python3/dist-packages/matplotlib/tests/test_backend_qt4.py
/usr/lib/python3/dist-packages/matplotlib/tests/test_axes.py
/usr/lib/python3/dist-packages/matplotlib/tests/test_backend_pdf.py
/usr/lib/python3/dist-packages/matplotlib/tests/test_pickle.py
/usr/lib/python3/dist-packages/matplotlib/tests/test_backend_pgf.py
/usr/lib/python3/dist-packages/matplotlib/tests/test_type1font.py
/usr/lib/python3/dist-packages/matplotlib/tests/__init__.py
/usr/lib/python3/dist-packages/matplotlib/tests/test_cycles.py
/usr/lib/python3/dist-packages/matplotlib/tests/test_backend_bases.py
/usr/lib/python3/dist-packages/matplotlib/tests/test_patches.py
/usr/lib/python3/dist-packages/matplotlib/tests/test_text.py
/usr/lib/python3/dist-packages/matplotlib/tests/test_triangulation.py
/usr/lib/python3/dist-packages/matplotlib/tests/test_basic.py
/usr/lib/python3/dist-packages/matplotlib/tests/test_simplification.py
/usr/lib/python3/dist-packages/matplotlib/tests/test_rcparams.py
/usr/lib/python3/dist-packages/matplotlib/tests/test_cbook.py
/usr/lib/python3/dist-packages/matplotlib/tests/mpltest.ttf
/usr/lib/python3/dist-packages/matplotlib/tests/test_mlab.py
/usr/lib/python3/dist-packages/matplotlib/tests/test_patheffects.py
/usr/lib/python3/dist-packages/matplotlib/tests/test_units.py
/usr/lib/python3/dist-packages/matplotlib/tests/test_colorbar.py
/usr/lib/python3/dist-packages/matplotlib/tests/test_offsetbox.py
/usr/lib/python3/dist-packages/matplotlib/tests/test_collections.py
/usr/lib/python3/dist-packages/matplotlib/tests/test_backend_qt5.py
/usr/lib/python3/dist-packages/matplotlib/tests/test_coding_standards.py
/usr/lib/python3/dist-packages/matplotlib/tests/test_labeled_data_unpacking.py
/usr/lib/python3/dist-packages/matplotlib/tests/test_agg.py
/usr/lib/python3/dist-packages/matplotlib/tests/test_lines.py
/usr/lib/python3/dist-packages/matplotlib/tests/test_dviread.py
/usr/lib/python3/dist-packages/matplotlib/tests/test_style.py
/usr/lib/python3/dist-packages/matplotlib/tests/test_image.py
/usr/lib/python3/dist-packages/matplotlib/tests/test_container.py
/usr/lib/python3/dist-packages/matplotlib/tests/test_spines.py
/usr/lib/python3/dist-packages/matplotlib/tests/test_ticker.py
/usr/lib/python3/dist-packages/matplotlib/tests/test_backend_svg.py
/usr/lib/python3/dist-packages/matplotlib/tests/test_gridspec.py
/usr/lib/python3/dist-packages/matplotlib/tests/test_sankey.py
/usr/lib/python3/dist-packages/matplotlib/tests/test_ttconv.py
/usr/lib/python3/dist-packages/matplotlib/tests/test_table.py
/usr/lib/python3/dist-packages/matplotlib/tests/test_arrow_patches.py
/usr/lib/python3/dist-packages/matplotlib/tests/test_skew.py
/usr/lib/python3/dist-packages/matplotlib/tests/test_delaunay.py
/usr/lib/python3/dist-packages/matplotlib/tests/test_animation.py
/usr/lib/python3/dist-packages/matplotlib/tests/test_quiver.py
/usr/lib/python3/dist-packages/matplotlib/tests/test_png.py
/usr/lib/python3/dist-packages/matplotlib/tests/test_artist.py
/usr/lib/python3/dist-packages/matplotlib/tests/test_widgets.py
/usr/lib/python3/dist-packages/matplotlib/tests/test_rcparams.rc
/usr/lib/python3/dist-packages/matplotlib/tests/test_mathtext.py
/usr/lib/python3/dist-packages/matplotlib/tests/test_backend_ps.py
/usr/lib/python3/dist-packages/matplotlib/tests/test_subplots.py
/usr/lib/python3/dist-packages/matplotlib/tests/test_path.py
/usr/lib/python3/dist-packages/matplotlib/tests/test_legend.py
/usr/lib/python3/dist-packages/matplotlib/tests/test_utf32_be_rcparams.rc
/usr/lib/python3/dist-packages/matplotlib/tests/test_colors.py
/usr/lib/python3/dist-packages/matplotlib/tests/test_tightlayout.py
/usr/lib/python3/dist-packages/matplotlib/tests/test_scale.py
/usr/lib/python3/dist-packages/matplotlib/tests/test_transforms.py
/usr/lib/python3/dist-packages/matplotlib/tests/test_texmanager.py
/usr/lib/python3/dist-packages/matplotlib/finance.py
/usr/lib/python3/dist-packages/matplotlib/delaunay
/usr/lib/python3/dist-packages/matplotlib/delaunay/testfuncs.py
/usr/lib/python3/dist-packages/matplotlib/delaunay/__init__.py
/usr/lib/python3/dist-packages/matplotlib/delaunay/triangulate.py
/usr/lib/python3/dist-packages/matplotlib/delaunay/interpolate.py
/usr/lib/python3/dist-packages/matplotlib/scale.py
/usr/lib/python3/dist-packages/matplotlib/rcsetup.py
/usr/lib/python3/dist-packages/matplotlib/streamplot.py
/usr/lib/python3/dist-packages/matplotlib/text.py
/usr/lib/python3/dist-packages/matplotlib/patheffects.py
/usr/lib/python3/dist-packages/matplotlib/container.py
/usr/lib/python3/dist-packages/matplotlib/backend_tools.py
/usr/lib/python3/dist-packages/matplotlib/fontconfig_pattern.py
/usr/lib/python3/dist-packages/matplotlib/_png.cpython-35m-x86_64-linux-gnu.so
/usr/lib/python3/dist-packages/matplotlib/spines.py
/usr/lib/python3/dist-packages/matplotlib/figure.py
/usr/lib/python3/dist-packages/matplotlib/axis.py
/usr/lib/python3/dist-packages/matplotlib/tight_layout.py
/usr/lib/python3/dist-packages/matplotlib/artist.py
/usr/lib/python3/dist-packages/matplotlib/colors.py
/usr/lib/python3/dist-packages/matplotlib/type1font.py
/usr/lib/python3/dist-packages/matplotlib/collections.py
/usr/lib/python3/dist-packages/matplotlib/font_manager.py
/usr/lib/python3/dist-packages/matplotlib/_qhull.cpython-35m-x86_64-linux-gnu.so
/usr/lib/python3/dist-packages/matplotlib/blocking_input.py
/usr/lib/python3/dist-packages/matplotlib/backend_bases.py
/usr/lib/python3/dist-packages/matplotlib/mlab.py
/usr/lib/python3/dist-packages/matplotlib/dates.py
/usr/lib/python3/dist-packages/matplotlib/patches.py
/usr/lib/python3/dist-packages/matplotlib/image.py
/usr/lib/python3/dist-packages/matplotlib/_contour.cpython-35m-x86_64-linux-gnu.so
/usr/lib/python3/dist-packages/matplotlib/ft2font.cpython-35m-x86_64-linux-gnu.so
/usr/lib/python3/dist-packages/matplotlib/sphinxext
/usr/lib/python3/dist-packages/matplotlib/sphinxext/__init__.py
/usr/lib/python3/dist-packages/matplotlib/sphinxext/mathmpl.py
/usr/lib/python3/dist-packages/matplotlib/sphinxext/only_directives.py
/usr/lib/python3/dist-packages/matplotlib/sphinxext/tests
/usr/lib/python3/dist-packages/matplotlib/sphinxext/tests/__init__.py
/usr/lib/python3/dist-packages/matplotlib/sphinxext/tests/tinypages
/usr/lib/python3/dist-packages/matplotlib/sphinxext/tests/tinypages/conf.py
/usr/lib/python3/dist-packages/matplotlib/sphinxext/tests/tinypages/index.rst
/usr/lib/python3/dist-packages/matplotlib/sphinxext/tests/tinypages/_static
/usr/lib/python3/dist-packages/matplotlib/sphinxext/tests/tinypages/_static/README.txt
/usr/lib/python3/dist-packages/matplotlib/sphinxext/tests/tinypages/some_plots.rst
/usr/lib/python3/dist-packages/matplotlib/sphinxext/tests/test_tinypages.py
/usr/lib/python3/dist-packages/matplotlib/sphinxext/plot_directive.py
/usr/lib/python3/dist-packages/matplotlib/quiver.py
/usr/lib/python3/dist-packages/matplotlib/afm.py
/usr/lib/python3/dist-packages/matplotlib/_mathtext_data.py
/usr/lib/python3/dist-packages/matplotlib/sankey.py
/usr/share
/usr/share/doc
/usr/share/doc/python3-matplotlib
/usr/share/doc/python3-matplotlib/copyright
/usr/share/doc/python3-matplotlib/changelog.Debian.gz
/usr/share/doc/python3-matplotlib/NEWS.Debian.gz
/usr/lib/python3/dist-packages/matplotlib/backends/web_backend/jquery/js/jquery-ui.min.js
/usr/lib/python3/dist-packages/matplotlib/backends/web_backend/jquery/js/jquery-1.7.1.min.js
/usr/lib/python3/dist-packages/matplotlib/backends/web_backend/jquery/css/themes/base/jquery-ui.min.css

**Comment by Jana-Wessels** [comment]
Fixed by going back to python 3.5
